cmd/dictionary-service/configuration.go

```diff
@@ -30,6 +43,6 @@ func makeConfigurationEndpoints(r *mux.Router, configurationService service.Conf
 	r.Methods("GET", "OPTIONS").Path("/api/configs/{day}").Handler(http.NewServer(
```

Comment:
Dla spójności można by było zmienić tutaj na **/api/config/{day}** lub w reszcie API na **/api/configs/****

Reply:
to jest endpoint, który zwraca listę - dla odróżnienia url od tego, który zwraca jeden element, musiałem coś mu dodać - `/api/config/{day}` już istnieje.